Cubical/Data/Sigma/Properties.agda

```diff
@@ -272,3 +164,43 @@ Iso.leftInv (Σ-ap-iso₂ isom) (x , y') = ΣPathP (refl , Iso.leftInv (isom x)
   → (Σ X Y)
   ≡ (Σ X' Y')
 Σ-ap'  {ℓ} {ℓ'} isom isom' = cong₂ (λ (a : Type ℓ) (b : a → Type ℓ') → Σ a λ x → b x) isom isom'
+
+-- Alternative version for path in Σ-types, as in the HoTT book
+
+ΣPathTransport : (a b : Σ A B) → Type _
+ΣPathTransport {B = B} a b =
+  Σ (fst a ≡ fst b) (λ p → transport (λ i → B (p i)) (snd a) ≡ snd b)
+
+_Σ≡T_ : (a b : Σ A B) → Type _
```

Comment:
I don't like this name... Why not just use \SigmaPathTransport as it's a bit more informative?